plugins/telemetry-otel/src/internalClusterTest/java/org/opensearch/telemetry/tracing/TelemetryTracerEnabledSanityIT.java

```diff
@@ -81,15 +83,28 @@ public void testSanityChecksWhenTracingEnabled() throws Exception {
             Arrays.asList(
                 new AllSpansAreEndedProperly(),
                 new AllSpansHaveUniqueId(),
-                new NumberOfTraceIDsEqualToRequests(),
+                new NumberOfTraceIDsEqualToRequests(Attributes.create().addAttribute("action", "indices:data/read/search[phase/query]")),
                 new TotalRootSpansEqualToRequests()
             )
         );
 
-        InMemorySingletonSpanExporter exporter = InMemorySingletonSpanExporter.create();
+        InMemorySingletonSpanExporter exporter = InMemorySingletonSpanExporter.INSTANCE;
         if (!exporter.getFinishedSpanItems().isEmpty()) {
-            validators.validate(exporter.getFinishedSpanItems(), 2);
+            /**
+             * At present, transport action is not instrumented and all the downstream search calls don't have a parent so,
+             * they all should be creating a separate trace. Once the transportAction will be instrumented this test will
+             * start failing and requires the value to be updated to 2.
+             */
+            validators.validate(exporter.getFinishedSpanItems(), 6);
         }
     }
 
+    private static void updateTelemetrySetting(Client client, boolean value) {
```

Comment:
You don't need to manipulate settings anymore, right?

Reply:
Yes, removing the same.